lib/utils.ts

```diff
@@ -84,7 +84,7 @@ export function difference<T>(setA: Set<T>, setB: Set<T>): Set<T> {
 
 export async function asCallback<T>(
 	promise: Promise<T>,
-	callback: (error: Error | void, value?: T) => void,
+	callback: (error: Error | void | null, value?: T) => void,
```

Comment:
Another place to drop `void`
```suggestion
	callback: (error: Error | null, value?: T) => void,
```

Reply:
dropped